src/main/java/frc/robot/subsystems/climb/ClimbWithXbox.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.climb;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Robot;
+
+public class ClimbWithXbox extends CommandBase {
+  DoubleSupplier power;
+  boolean isClimb;
+
+  /**
+   * Use this class to higher the lift for the hook to climb.
+   * 
+   * @param power   the power to give the motors
+   * @param isClimb whether to send power to the climb motors or the hook motors.
+   *                If True set the climb motors to the power.
+   */
+  public ClimbWithXbox(DoubleSupplier power, boolean isClimb) {
+    addRequirements(Robot.climb);
```

Comment:
addRequirements(Robot.climb);
bad! because we want to run hook and climb! insert two DoubleSupplier for climb and hook